docs/source/guide/upgrading.rst

```diff
@@ -4,7 +4,7 @@ Upgrading Notes
 
 Notes to refer to when upgrading ``boto3`` versions.
 
-1.4.0
+1.4.x
```

Comment:
So I ended up just changing it back to 1.4.0 and made a note that you had to use a version higher than 1.4.1 to use the option. I still want to include this under the 1.4.0 as beginning at that version was when transfer used threads by default as opposed to it being in-determinant.